keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -578,10 +578,13 @@ Compared to labels, attributes have values of exactly one type. Quantities are d
 in the separate `capacity` map. As described later on, both sets can be used in CEL expressions to select a
 specific resource for allocation on a node.
 
-To avoid any future conflicts, we reserve any attributes with the ".k8s.io/" domain prefix
-for future use and standardization by Kubernetes. This could be used to describe
-topology across resources from different vendors, for example, but this is out-
-of-scope for now.
+We are reserving the `kubernetes.io/` domain (and subdomains) prefix for
+attributes and capacities for standardization by the Kubernetes project. This
+reservation allows us to define common attributes that can describe hardware
+characteristics across resources from different vendors. Currently, we are
+defining one such standard attribute: `kubernetes.io/pcieRoot`. Details on its
```

Comment:
Standardizing attributes seems very useful. Could we have a section specifically talk about standardized attributes? For example, could driverVersion, runtimeVersion be standardized, too?

Reply:
Yes it's okay to do that in a separate PR. The idea is to standardize common attributes to improve the usability and consistency. This would be somewhat similar to well-known labels https://kubernetes.io/docs/reference/labels-annotations-taints/